ui/app/models/pki/action.js

```diff
@@ -23,6 +24,8 @@ const validations = {
 export default class PkiActionModel extends Model {
   @service secretMountPath;
 
+  @tracked actionType; // used to toggle between different form fields when creating configuration
```

Comment:
Needed to move this to the model for conditional validation since `issuerName` is not a required field when generating a CSR.